tests/migration/online_migration/zypper_migration.pm

```diff
@@ -14,21 +14,17 @@ use testapi;
 use serial_terminal 'select_serial_terminal';
 use utils;
 use power_action_utils 'power_action';
-use version_utils qw(is_desktop_installed is_sles4sap is_leap_migration is_sle_micro);
+use version_utils qw(is_desktop_installed is_sles4sap is_leap_migration is_sle_micro verify_os_version);
 use Utils::Backends 'is_pvm';
 use Utils::Logging 'upload_solvertestcase_logs';
 use transactional;
 
-sub check_migrated_version {
-    # check if the migration success or not by checking the /etc/os-release file with the VERSION
-    my $target_version = get_var("TARGET_VERSION", get_required_var("VERSION"));
-    assert_script_run("grep VERSION= /etc/os-release | grep $target_version");
-}
-
 sub run {
     my $self = shift;
     select_console 'root-console';
-
+    # This multi-naming of the same var should be addressed in poo#157873
+    my $version_to_verify = is_sle_micro ? get_var("FROM_VERSION") : get_var(("HDDVERSION"), get_required_var("ORIGIN_SYSTEM_VERSION"));
```

Comment:
Breakage is introduced in this line.

The test is doing an rolling migration of a cluster node. Systems start on 15-SP{n} and get migrated by `zypper_migration` to 15-SP{n+1}. As such, when `verify_os_version` is called, the actual OS version is `ORIGIN_SYSTEM_VERSION` not `HDDVERSION` but both are set.

Reply:
I am removing the verify_os_version as there are many tests that use different var naming for the same thing